packages/next-optimized-images/lib/index.js

```diff
@@ -23,6 +23,8 @@ const withOptimizedImages = (nextConfig = {}, nextComposePlugins = {}) => {
     overwriteImageLoaderPaths,
   } = getConfig(nextConfig)
 
+  nextConfig.__nextOptimizedImages = true
```

Comment:
Only added this so I could assert on the application of `next-optimized-images` without mocking. Module mocking isn't quite ready for primetime when testing native ESM.